### PR TITLE
CSI-3091 bump driver version to 1.7.0

### PR DESCRIPTION
**Pull request by zingero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Use the following steps to install the operator and driver, with [GitHub](https:
     1. Download the manifest from GitHub.
 
         ```
-        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.6.0/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.7.0/deploy/installer/generated/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
```
Comment (oriyarde) [inline README.md:110]: I think the README should be updated right before GA, so it won't be broken most of the time.
@ArbelNathan what do you think?

Comment (ArbelNathan) [reply README.md:110]: same, readme update is a separate ticket for GA. right now we want it to be true, especially when `develop` is the default branch and that the readme that everyone sees.

Comment (zingero) [reply README.md]: Done

File: controller/controller_server/csi_controller_server.py [hunk 2]
```diff
@@ -47,10 +48,8 @@ def CreateVolume(self, request, context):
         try:
             utils.validate_create_volume_request(request)
         except ObjectIdError as ex:
-            logger.exception(ex)
-            context.set_details(ex.message)
-            context.set_code(grpc.StatusCode.NOT_FOUND)
-            return csi_pb2.CreateVolumeResponse()
+            return handle_exception(ex, context, grpc.StatusCode.NOT_FOUND,
+                                    csi_pb2.CreateVolumeResponse)
```
Comment (oriyarde) [inline controller/controller_server/csi_controller_server.py:52]: I don't see how this is related to the PR

Comment (zingero) [reply controller/controller_server/csi_controller_server.py:52]: It came with `git pull -r`

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**